scripting/include/srccoop/classdef.inc

```diff
@@ -892,6 +1019,62 @@ methodmap CBasePlayer < CBaseCombatCharacter
 	{
 		SetEntProp(this.GetEntIndex(), Prop_Send, "m_bWearingSuit", bShouldHaveSuit);
 	}
+	public void PlayAdmireGloves(const bool bStartAnim)
```

Comment:
I'd probably move this to playerpatch. It's a bit long and variables here are not shared between plugins anyways.